api/test/integration/common.yaml

```
@@ -22,6 +22,7 @@ variables:
   restart_delay_cluster: 150
   upgrade_delay: 60
   global_db_delay: 30
+  cluster_sync_delay: 20
```

This value is too high. The cluster synchronization should last 10 seconds at worst in the testing environment. To save some time, this should be reduced to `12`. A 30% margin must be enough. 
```suggestion
  cluster_sync_delay: 12
```